### DISCUSSION TITLE
Multi-user on Roku

### DISCUSSION THREAD

**Discussion by lorodoes** [jellyfin-archive/jellyfin-roku-legacy#1540] [Q&A] [answered]
I know this might be a server side thing, but I'm not to sure. So, I started at the client. I would full switch to jellyfin in heart beat if I could quick select between users. Even if I had to have my phone to verify it was me that's fine, but I don't want to have to set everything up again each time and re-enter in all information. There has to be a method to make user switching easy and fairly secure. The main reason I want this is to be able to switch profiles between my kids, my wife and myself for watch history and stuff like that. Am I missing something in the roku client?

**Comment by michaelcresswell** [comment, marked as answer]
Simplified user switching has been requested for some time, going back to at least #310.

Fortunately, @cewert has written a Change User option for the home screen (#1374). This has been merged into unstable so it should be included in the next Roku client release.

**Reply by ebits21** [reply]
Yep check it out in 2.0.0 😊
